alot/widgets/thread.py

```diff
@@ -22,6 +24,13 @@ class MessageSummaryWidget(urwid.WidgetWrap):
     one line summary of a :class:`~alot.db.message.Message`.
     """
 
+    pat_blank_line = re.compile(r'^\s*$')
+    pat_quote_header_line = re.compile(
+        r'^(?:On .*, .* wrote:|----- Original Message -----)'
```

Comment:
This is going to be problematic. It first assumes that the message will be in english (which, I get plenty where these sort of metadata tags are not), second it assumes the form of the tag, which may be okay, but I'm not sure.

Reply:
Fair point. I'm thinking that a lot of the parameters would need to be configurable: the exclusion patterns, threshold values, etc. This would allow the user to tweaking things to fit their typical email.

I'm unsure how alot's config parser would react to regex patterns, hopefully it's doable.

Reply:
alot uses [configobj](http://www.voidspace.org.uk/python/configobj.html) for configuration. I don't think it will have a problem with regex, but we probably need to `re.escape()` the values when they go into alot's config object.